.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Nodejs
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          always-auth: true
+          registry-url: 'https://registry.npmjs.org'
+      - name: Build and Publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
```

Comment:
```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Reply:
@clearloop secret name is `NPM_AUTH_TOKEN` not `NODE_AUTH_TOKEN`

Reply:
XDD got into mess rn, let me try to replace it back